lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -234,6 +238,23 @@ export class ResponseHandler {
                 );
             }
 
+            if (
```

Comment:
There's some duplicate logic here I'll most likely refactor this before merging.